components/Tweet/Tweet.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { TwitterShareButton } from "react-twitter-embed"
+
+const Tweet = () => {
```

Comment:
Where is this used?

Reply:
Ahh.  It's not.  I was going to use it but didn't.  I will remove.